### DISCUSSION TITLE
A small request about Azata

### DISCUSSION THREAD

**Discussion by q1421813533** [Truinto/DarkCodex#129] [Ideas]
Hello, Truinto.
The butterflies flying around the Azata always disappear when the team go to another place. Could you add a feature to fix this bug?
Thanks in advance.

**Comment by Truinto** [comment]
That's really difficult. I assume it's treated as a familiar. These already have logic to spawn them when changing areas. So it might be specific areas or specific transitions that are bugged. In any case there is a reason why it doesn't work in the first place and I am not smarter than Owlcat's devs.

Btw, I think familiars don't spawn in the capital.

**Comment by Truinto** [comment]
Puh, that sounds actually really difficult. SFX are not my speciality. I already have issues with just accessing SFX to pause some during cutscenes.
